contrib/passport_guide.lua

```diff
@@ -0,0 +1,73 @@
+--[[
+PASSPORT CROPPING GUIDE
+guides for cropping passport photos based on documents from the Finnish police
+(https://www.poliisi.fi/instancedata/prime_product_julkaisu/intermin/embeds/poliisiwwwstructure/38462_Passikuvaohje_EN.pdf) describing passport photo dimensions of 47x36 mm and 500x653 px for digital biometric data stored in passports. They use ISO 19794-5 standard based on ICAO 9303 regulations which should also be compliant for all of Europe.
+
+AUTHOR
+Kåre Hampf (k.hampf@gmail.com)
+
+INSTALLATION
+* copy this file in $CONFIGDIR/lua/ where CONFIGDIR is your darktable configuration directory
+* add the following line in the file $CONFIGDIR/luarc
+  require "passport_guide"
+* (optional) add the line:
+  "plugins/darkroom/clipping/extra_aspect_ratios/passport 36x47mm=47:36"
+  to $CONFIGDIR/darktablerc
+
+USAGE
+* when using the cropping tool, select "passport" as guide and if you added the line in yout rc
+  select "passport 36x47mm" as aspect
+
+LICENSE
+GPLv2
+
+]]
+
+local dt = require "darktable"
+dt.configuration.check_version(...,{2,0,0},{3,0,0},{4,0,0})
+
+dt.guides.register_guide("passport",
+-- draw
+function(cr, x, y, w, h, zoom_scale)
```

Comment:
Please test if you can declare this function as local. 
> local function(cr, x, y, w, h, zoom_scale)
Extra bonus for more meaningful names. x and y are fine, but what is cr? Replacing w and h would be nice but not that important.